website/server/libs/chat/group-chat.js

```diff
@@ -35,73 +35,76 @@ export function translateMessage (lang, info) {
 
   switch (info.type) {
     case 'quest_start':
-      msg = `\`${shared.i18n.t('chatQuestStarted', {questName: questScrolls[info.quest].text(lang)}, lang)}\``;
+      msg = `${shared.i18n.t('chatQuestStarted', {questName: questScrolls[info.quest].text(lang)}, lang)}`;
```

Comment:
this looks good but here like in other messages below given that the string is only the result of the i18n.t call it's not necessary to use the backslashes, so

> msg = `${shared.i18n.t('chatQuestStarted', {questName: questScrolls[info.quest].text(lang)}, lang)}`;

can be

> msg = shared.i18n.t('chatQuestStarted', {questName: questScrolls[info.quest].text(lang)}, lang);